libminifi/CMakeLists.txt

```diff
@@ -48,6 +49,19 @@ file(GLOB SPD_SOURCES "../include/spdlog/*")
 add_library(spdlog STATIC ${SPD_SOURCES})
 add_library(minifi STATIC ${SOURCES})
 
+# Include Boost System
+find_package(Boost COMPONENTS system REQUIRED)
```

Comment:
There seemed to be a little hesitation about using boost. My guess is that -Os added to flags won't actually cause the expected increase; however, it does potentially cause some bloat.